src/components/SectionAbout/SectionAboutUs.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @next/next/no-img-element */
+import './sectionAboutUs.scss';
+
+export const SectionAboutUs: React.FC = () => {
+  return (
+    <section>
+      <div className="sectionAboutUs__container container">
+        <h2 className="section-title sectionAboutUs-title">About us</h2>
```

Comment:
"section-title" to nie bo BEMowemu. Zrób po BEMowemu

Reply:
to section-title to jest globalny styl dla sekcji